src/creatures/players/player.cpp

```diff
@@ -7712,3 +7712,25 @@ std::shared_ptr<Container> Player::getLootPouch() {
 
 	return container;
 }
+
+namespace {
+	std::unordered_set<ConditionType_t> allowedConditions = {
+		CONDITION_ENERGY,
+		CONDITION_FIRE,
+		CONDITION_POISON,
+		CONDITION_BLEEDING
```

Comment:
Suggested by: Gtravisani
```suggestion
		CONDITION_BLEEDING,
		CONDITION_CURSED,
    CONDITION_DAZZLED
```